router/config.go

```diff
@@ -647,6 +648,8 @@ func LoadConfigWithOptions(path string, loadIdentity bool) (*Config, error) {
 
 	cfg.Metrics.ReportInterval = time.Minute
 	cfg.Metrics.MessageQueueSize = 10
+	cfg.Metrics.EventQueueSize = 256
```

Comment:
i'd really like to see these as actual defaults/consts somewhere. i know it's somewhat silly but it's nice to see them all in one place. also it'll be useful if we ever decide to add these to the configuration generator, which I'd really like to see us move to some day. maybe it's a nit but it'd be nice to have a list of these constants in one place imo

Reply:
I'll worry about it when we figure out how to make the config generator. It's very likely that we won't use constants for that anyway. It depends on what's easiest for a code generator to read.

Reply:
? we have a config generator. i'd like to see us use it more

Reply:
Sorry, I was thinking config doc generator. AFAIK the config generator doesn't use any introspection. I'd like to have some sort of metadata that could be used for both, but I suspect that consts would be tricky to use in that way.